compiler/rustc_interface/src/interface.rs

```diff
@@ -382,6 +382,12 @@ pub fn run_compiler<R: Send>(config: Config, f: impl FnOnce(&Compiler) -> R + Se
     // Set parallel mode before thread pool creation, which will create `Lock`s.
     rustc_data_structures::sync::set_dyn_thread_safe_mode(config.opts.unstable_opts.threads > 1);
 
+    let handler = EarlyErrorHandler::new(config.opts.error_format);
+    // Check jobserver before run_in_thread_pool_with_globals, which call jobserver::acquire_thread
```

Comment:
```suggestion
    // Initialize jobserver before run_in_thread_pool_with_globals, which call jobserver::acquire_thread
```
Could you rename `jobserver::check` into `jobserver::initialize_checked` so it would be more obvious that it's something mandatory?